### DISCUSSION TITLE
Strapi 实现用户注册与登录 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#163] [General]
# Strapi 实现用户注册与登录 - 愧怍的小站

在官方博客 Registration and Login (Authentication) with Vue.js and Strapi 中演示如何实现注册与登录。实际重点部分是 Strapi 的角色和权限插件，可以说这个插件让开发者不用再为项目考虑的用户登录注册与鉴权相关。

https://kuizuo.cn/strapi-user-register-and-login#comments

**Comment by sunzehui** [comment]
这个东西想扩充API就得用全局的strapi实例到处get，代码提示几乎没有，巨坑！

**Reply by kuizuo** [reply]
其实可以上ts的，不过要扩展确实要写一部分的代码。不过我目前也只是尝鲜使用，目前国内strapi使用确实不容乐观。
